packages/amplify_api/ios/Classes/FlutterApiErrorHandler.swift

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: I'm not sure we actually need to change these license years unless it's a brand new file?

Reply:
Ah gotcha - okay I'll change it back

Reply:
Actually, I think it's required as part of SwiftFormat to match the template, which has 2021. I can change it to match [amplify-ios](https://github.com/aws-amplify/amplify-ios/blob/main/.swiftformat) which doesn't have the date or just remove that from SwiftFormat altogether. Lemme know.

Reply:
Oh I see - I think it's ok. We might need to re-discuss before the end of year.